data/ecosystems/p/polkadot.toml

```diff
@@ -2,244 +2,389 @@
 title = "Polkadot"
 
 sub_ecosystems = [
```

Comment:
Many of these subecosystems need to be added in their own toml files for the build to succeed.  Can you please add those in separate PRs and then do one PR to add them all to Polkadot to make for easier review?